pkg/genericclioptions/clusteradm_flags.go

```diff
@@ -23,4 +24,5 @@ func NewClusteradmFlags(f cmdutil.Factory) *ClusteradmFlags {
 func (f *ClusteradmFlags) AddFlags(flags *pflag.FlagSet) {
 	flags.BoolVar(&f.DryRun, "dry-run", false, "If set the generated resources will be displayed but not applied")
 	flags.IntVar(&f.Timeout, "timeout", 300, "extend timeout from 300 secounds ")
+	flags.StringVar(&f.Context, "context", "", "The name of the kubeconfig context to use")
```

Comment:
`kubectl` have already provide a flag `--context` to specify which context to use, maybe you can't use the flag `context` here? wondering whether this will override the kubectl flag? or do we need to do something to handle this?

Reply:
> `kubectl` have already provide a flag `--context` to specify which context to use, maybe you can't use the flag `context` here? wondering whether this will override the kubectl flag? or do we need to do something to handle this?

I did not think about that before. I think we can get context from `kubectl` flag `--context`.